eng/Versions.props

```diff
@@ -5,8 +5,7 @@
     <MinorVersion>0</MinorVersion>
     <PatchVersion>0</PatchVersion>
     <VersionPrefix>$(MajorVersion).$(MinorVersion).$(PatchVersion)</VersionPrefix>
-    <PreReleaseVersionLabel Condition="'$(SuppressFinalPackageVersion)' != 'true'">rc.2</PreReleaseVersionLabel>
-    <PreReleaseVersionLabel Condition="'$(PreReleaseVersionLabel)' == ''">preview.5</PreReleaseVersionLabel>
+    <PreReleaseVersionLabel>preview.5</PreReleaseVersionLabel>
```

Comment:
We no longer need the distinction as packages will be stable or preview 5 depending or not if their stable version is being suppressed